app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -35,10 +45,7 @@ class MainActivity : AppCompatActivity() {
     private fun setupBottomNavigationBar() {
         val bottomNavView = findViewById<BottomNavigationView>(R.id.bottom_nav)
 
-        val navController = Navigation.findNavController(
-            this,
-            R.id.nav_host_fragment
-        )
+        val navController = Navigation.findNavController(this, R.id.nav_host_fragment)
```

Comment:
Put all on a single line because its easier to read and isn't too long

Reply:
That can be a matter of opinion, or programmers preference.
For reformatting... 
1) A caution.  Significant reformats can make future commit diffs difficult, as git has to deal with significant reordering of the logic.
2) The IDE can do this automatically, and you can provide a template for the IDE to use.  Ensure all developers are using this same template, or the one that comes by default with the IDE, and have them reformat before committing.  This ameliorates issue 1, above.